### PR TITLE
Target 10.7

### PR DESCRIPTION
**Pull request by crobibero**
Follows https://github.com/jellyfin/jellyfin-plugin-emailnotifications/pull/17

### CODE REVIEW

**Comment by oddstr13** [comment]
Rebase needed

File: MediaBrowser.Plugins.SmtpNotifications/Api/EmailNotificationController.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿using System;
```
Comment (oddstr13) [inline MediaBrowser.Plugins.SmtpNotifications/Api/EmailNotificationController.cs:1]: Incomplete rename? Should `SmtpNotificationsController.cs` not be deleted as a counterpart to the addition of this file?

**Review by oddstr13 — APPROVED** [review]

**Comment by crobibero** [comment]
I think I did the API migration twice, but the end result seems to be fine

**Comment by oddstr13** [comment]
Looked kinda like you accidentally marked to keep the file during the merge actually :man_shrugging:

File: MediaBrowser.Plugins.SmtpNotifications/Api/EmailNotificationController.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        {
+            await new Notifier(_notifierLogger).SendNotification(
+                new UserNotification
+                {
+                    Date = DateTime.UtcNow,
+                    Description = "This is a test notification from Jellyfin Server",
+                    Level = Model.Notifications.NotificationLevel.Normal,
+                    Name = "Test Notification",
+                    User = _userManager.GetUserById(userId)
+                }, CancellationToken.None);
```
Comment (BaronGreenback) [inline MediaBrowser.Plugins.SmtpNotifications/Api/EmailNotificationController.cs:55]:
```suggestion
            await new Notifier(_notifierLogger).SendNotification(
                new UserNotification
                {
                    Date = DateTime.UtcNow,
                    Description = "This is a test notification from Jellyfin Server",
                    Level = Model.Notifications.NotificationLevel.Normal,
                    Name = "Test Notification",
                    User = _userManager.GetUserById(userId)
                }, CancellationToken.None).ConfigureAwait(false);
```

**Review by BaronGreenback — CHANGES_REQUESTED** [review]

**Review by BaronGreenback — APPROVED** [review]

File: MediaBrowser.Plugins.SmtpNotifications/Api/EmailNotificationController.cs (hunk 2 above)
Comment (oddstr13) [reply MediaBrowser.Plugins.SmtpNotifications/Api/EmailNotificationController.cs]: Is there any linting we can apply for this?

**Pull request merged**